### PR TITLE
Added documentation for placeholder components

### PR DESCRIPTION
**Pull request by fraincs**
Added documentation for : 
- Header
- Footer
- Section
- Content

### CODE REVIEW

File: apps/docs/content/components/placeholders/Content.mdx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    TODO: Example of passing custom childrens to the components to fake a slot
+
+    ### Custom Component
+
+    TODO: Example of creating a custom version of this component
+</FeatureFlag>
+
+## Props
+
+### Content
```
Comment (vicky-comeau) [inline apps/docs/content/components/placeholders/Content.mdx:48]: Why have a sub title here? we know it is the props for Content, we don't do this elsewhere unless there are multiple component props on one page.

File: apps/docs/content/components/placeholders/Footer.mdx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    TODO: Example of passing custom childrens to the components to fake a slot
+
+    ### Custom Component
+
+    TODO: Example of creating a custom version of this component
+</FeatureFlag>
+
+## Props
+
+### Footer
```
Comment (vicky-comeau) [inline apps/docs/content/components/placeholders/Footer.mdx:48]: same comment as above for the sub title

File: apps/docs/content/components/placeholders/Header.mdx [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
+title: Header
+description: A placeholder for an header section.
```
Comment (vicky-comeau) [inline apps/docs/content/components/placeholders/Header.mdx:3]:
```suggestion
description: A placeholder for a header section.
```

File: apps/docs/content/components/placeholders/Footer.mdx [hunk 4]
```diff
@@ -0,0 +1,50 @@
+---
+title: Footer
+description: A placeholder for an footer section.
```
Comment (vicky-comeau) [inline apps/docs/content/components/placeholders/Footer.mdx:3]:
```suggestion
description: A placeholder for a footer section.
```

File: apps/docs/content/components/placeholders/Section.mdx [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    TODO: Example of passing custom childrens to the components to fake a slot
+
+    ### Custom Component
+
+    TODO: Example of creating a custom version of this component
+</FeatureFlag>
+
+## Props
+
+### Section
```
Comment (vicky-comeau) [inline apps/docs/content/components/placeholders/Section.mdx:48]: same as above

File: packages/components/src/Avatar/src/AnonymousAvatar.temp.tsx
Comment (vicky-comeau) [inline packages/components/src/Avatar/src/AnonymousAvatar.temp.tsx:1]: why was this added?

File: apps/docs/content/components/placeholders/Section.mdx [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+    TODO: If we have some guidelines about this component&apos;s usage
+
+    ### Accessibility ?
+
+    TODO: If we have some guidelines about this component and accessibility
+</FeatureFlag>
+
+<FeatureFlag flag="rc">
+    ## Anatomy
```
Comment (vicky-comeau) [inline apps/docs/content/components/placeholders/Section.mdx:22]: We can add composed components for this one.
which would be Header and ListBoxItem

File: apps/docs/content/components/placeholders/Content.mdx [hunk 7]
```diff
@@ -0,0 +1,48 @@
+---
+title: Content
+description: A placeholder for the main content section of a component.
+category: "placeholders"
+links:
+    source: https://github.com/gsoft-inc/wl-hopper/blob/main/packages/components/src/layout/src/Content.tsx
+---
+
+<Example src="layout/docs/content/preview" isOpen />
```
Comment (alexasselin008) [inline apps/docs/content/components/placeholders/Content.mdx:9]: This is really too thin of a documentation. 

you should add what is in orbiter  like thge description: 

A content placeholder component provides no specific styling by itself, but receives styling from the parent container. In addition, a content placeholder will be automatically placed within the container's layout according to Orbiter guidelines.

Comment (fraincs) [reply apps/docs/content/components/placeholders/Content.mdx:9]: The thing is that it can't be the description as it'll be too long in the component preview page, and adding this before the example opens the door to having to do the same for all components, which might not ideal.

Comment (alexasselin008) [reply apps/docs/content/components/placeholders/Content.mdx:9]: Can you try to figure something out? As of right now we added a page of doc to link to, but the doc page is useless

**Review by alexasselin008 — APPROVED** [review]
